src/ant/src/AntennaChecker.tcl

```diff
@@ -30,12 +30,12 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 sta::define_cmd_args "check_antennas" { [-verbose]\
-                                          [-net net]}
+                                          [-net net]};# checker off
 
 proc check_antennas { args } {
   sta::parse_key_args "check_antennas" args \
     keys {-report_file -net} \
-    flags {-verbose -report_violating_nets}
+    flags {-verbose -report_violating_nets};# checker off
```

Comment:
Why is the checker off here?

Reply:
Checker is off because of the deprecated arguments in `check_antennas`. As such some flags/keys will exist in proc but not help.